packages/@ionic/cli/src/commands/deploy/capacitor.ts

```diff
@@ -0,0 +1,66 @@
+import { BaseConfig } from "@ionic/cli-framework";
```

Comment:
Is this file temporary until the support of Cap 3 is merged?

Reply:
Yes, exactly.